CMakeLists.txt

```diff
@@ -293,7 +293,12 @@ if(AVIF_CODEC_LIBGAV1)
     )
 
     if(AVIF_LOCAL_LIBGAV1)
-        set(LIB_FILENAME "${CMAKE_CURRENT_SOURCE_DIR}/ext/libgav1/build/libgav1${CMAKE_STATIC_LIBRARY_SUFFIX}")
+        set (AVIF_LIBGAV1_BUILD_DIR "${CMAKE_CURRENT_SOURCE_DIR}/ext/libgav1/build")
+        # If ${ANDROID_ABI} is set, look for the library under that subdirectory.
```

Comment:
Is it true that we must pass -DANDROID_ABI=xxx to cmake when we build libavif with the Android NDK? This webpage says ANDROID_ABI is a required argument:

https://developer.android.com/ndk/guides/cmake#android_abi

Reply:
Yes that's correct. In the subsequent pull request, where we actually make use of this in libavif's android JNI, that argument will be set by the android gradle build system automatically.